message published doesnt have timestamp in properties.

immnk opened the pull request:
there is no property in config.js for enabling timestamp with the message so the brokers can read and send to the clients subscribing the brokers. Specifically to rabbitmq broker.

mcollina committed:
Added cors!

mcollina committed:
Added POST to update a resource.

mcollina committed:
Merge branch 'cors'

mcollina committed:
Rename test to avoid _spec.

mcollina committed:
NPM-friendly images.

mcollina committed:
Bumped v0.0.10.

campeterson committed:
Update README.md

mcollina committed:
Merge pull request #3 from campeterson/patch-1

Update README.md

mcollina committed:
Added db/ in .gitignore.

mcollina committed:
Bumped all dependencies.

mcollina committed:
Fixed MQTT wildcard configuration.

mcollina committed:
Updated copyright year.

mcollina committed:
Better host parameters. Automatic IPv6 detection for CoAP.

mcollina committed:
Added Mongo configuration to README.

mcollina committed:
Added configuration data for Redis.

mcollina committed:
Added MQTT configuration data.

mcollina committed:
Bumped v0.0.11.

mcollina committed:
Updated dependencies.

mcollina committed:
Bumped v0.0.12.

healarconr committed:
Add unit tests for the authentication and authorization callbacks. Update README.md with the proposed callback configuration.

healarconr committed:
Make new CoAP unit tests pass

Add missing done calls to the new CoAP unit tests

Provide some examples for the CoAP authentication and authorization callbacks in the README.md

healarconr committed:
Make new HTTP unit tests pass

Update the configuration description of the authentication and authorization callbacks in the README.md

healarconr committed:
Correct the new unit tests of the MQTT server and make them pass

mcollina committed:
Merge pull request #12 from healarconr/master

Authentication and authorization callbacks

mcollina committed:
Bumped 0.0.13.

diorahman committed:
Allow to serve libraries from hidden directories

The st module denies to serve libraries from hidden directories unless
the dot option is set to true. This condition is relevant when node is
installed using nvm and ponte is installed globally.

Signed-off-by: Dhi Aurrahman <diorahman@rockybars.com>

Commit:
Fix http payload and update dependencies

Before that fix the payload was persisted as string in a mongodb, while it
should be persisted as binary array.

Signed-off-by: Mirko Geissler <mirko.geissler@artcom.de>

Commit:
Fix crash in case of empty payload.

This is needed to allow removing retained topics with PUT.

Signed-off-by: Mirko Geissler <mirko.geissler@artcom.de>

mcollina committed:
Updated dependencies.

mcollina commented:
You should accept the eclipse CLA https://dev.eclipse.org/eclipse-webhook/services/status_details.php?id=5594c2bb12bcd.

But it's not clear the purpose of this PR.

mcollina committed:
Added links to the README.

mcollina committed:
Updated dependencies.

mcollina committed:
Merge pull request #13 from rockybars/public-libraries-inside-dot-dir

Allow to serve libraries from hidden directories

mcollina committed:
Bumped 0.0.14.

Commit:
Bumped dependencies, added Node4 to travis

Signed-off-by: Peter Sorowka <sorowka@skybus.io>

Commit:
Bumped Mosca and Ascoltatori, running on node4+

- adapted to new ascoltatori interface
- improved debug output in error case
- moved mqtt to dev dependencies
- added node4 and 5 to travis

Signed-off-by: Peter Sorowka <sorowka@skybus.io>

Commit:
Bumped Dependencies

- mosca@1.0.1
- ascoltatori@2.0.1
- node-coap@0.13.0

Signed-off-by: Peter Sorowka <sorowka@skybus.io>

mcollina committed:
Bumped all the other deps.

mcollina committed:
Fix link to website.

Closes #20.

mcollina committed:
Bumped 0.0.15.

psorowka committed:
Added 'use strict' keyword to all files

- fixed an accidental use of global var
- thereby fixed race condition on simultaneous HTTP requests
- fixed to cases of wrongly assigned callback functions
- bumped dependencies
- Restricted travis to node 5.5 and 5.6
- added node version comment to readme

Signed-off-by: Peter Sorowka <sorowka@cybus.io>

mcollina committed:
Merge pull request #22 from cybusio/fix-http-race-condition

Added 'use strict' keyword to all files

mcollina committed:
Bumped v0.0.16.

chombium committed:
refactoring to conform the JavaScript Standard Style Guide  

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

chombium committed:
dependancies upgrade and js standard to the precommit hook

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

chombium committed:
code cleanup removed the jshint configuration file

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

chombium committed:
Added initial JSDoc. More to come with #37

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

chombium committed:
Added eclipse project specific files to the ignores

Signed-off-by: Jovan Kostovski <chombium@gmail.com>

mcollina committed:
Merge pull request #41 from chombium/node-lts-refactoring

Node v6.9.5 refactoring and new code formatting
